### ISSUE TITLE
`wgpuAdapterHasFeature` symbol missing in v0.10.4.1

### ISSUE THREAD

**Issue by chances** [gfx-rs/wgpu-native#156]
macOS
Release: https://github.com/gfx-rs/wgpu-native/releases/tag/v0.10.4.1

The [`wgpuAdapterHasFeature`](https://github.com/webgpu-native/webgpu-headers/blob/4b4dbc3253ba03a1a7b7c42e18cead9dfc1142b8/webgpu.h#L1216) symbol is missing:

```
Undefined symbols for architecture x86_64:
  "_wgpuAdapterHasFeature", referenced from:
[sic]
```

**Comment by fcvarela** [comment]
These symbols are all missing from releases:

```  "_wgpuAdapterEnumerateFeatures", referenced from:
  "_wgpuAdapterHasFeature", referenced from:
  "_wgpuBufferGetConstMappedRange", referenced from:
  "_wgpuCommandEncoderFillBuffer", referenced from:
  "_wgpuCommandEncoderInsertDebugMarker", referenced from:
  "_wgpuCommandEncoderPopDebugGroup", referenced from:
  "_wgpuCommandEncoderPushDebugGroup", referenced from:
  "_wgpuCommandEncoderResolveQuerySet", referenced from:
  "_wgpuCommandEncoderWriteTimestamp", referenced from:
  "_wgpuComputePassEncoderBeginPipelineStatisticsQuery", referenced from:
  "_wgpuComputePassEncoderEndPipelineStatisticsQuery", referenced from:
  "_wgpuComputePassEncoderInsertDebugMarker", referenced from:
  "_wgpuComputePassEncoderPopDebugGroup", referenced from:
  "_wgpuComputePassEncoderPushDebugGroup", referenced from:
  "_wgpuComputePipelineGetBindGroupLayout", referenced from:
  "_wgpuComputePipelineSetLabel", referenced from:
  "_wgpuCreateInstance", referenced from:
  "_wgpuDeviceCreateComputePipelineAsync", referenced from:
  "_wgpuDeviceCreateQuerySet", referenced from:
  "_wgpuDeviceCreateRenderBundleEncoder", referenced from:
  "_wgpuDeviceCreateRenderPipelineAsync", referenced from:
  "_wgpuDeviceDestroy", referenced from:
  "_wgpuDeviceEnumerateFeatures", referenced from:
  "_wgpuDeviceHasFeature", referenced from:
  "_wgpuDevicePopErrorScope", referenced from:
  "_wgpuDevicePushErrorScope", referenced from:
  "_wgpuDeviceSetDeviceLostCallback", referenced from:
  "_wgpuDeviceSetUncapturedErrorCallback", referenced from:
  "_wgpuGetProcAddress", referenced from:
  "_wgpuInstanceProcessEvents", referenced from:
  "_wgpuQuerySetDestroy", referenced from:
  "_wgpuQueueOnSubmittedWorkDone", referenced from:
  "_wgpuRenderBundleEncoderDraw", referenced from:
  "_wgpuRenderBundleEncoderDrawIndexed", referenced from:
  "_wgpuRenderBundleEncoderDrawIndexedIndirect", referenced from:
  "_wgpuRenderBundleEncoderDrawIndirect", referenced from:
  "_wgpuRenderBundleEncoderFinish", referenced from:
  "_wgpuRenderBundleEncoderInsertDebugMarker", referenced from:
  "_wgpuRenderBundleEncoderPopDebugGroup", referenced from:
  "_wgpuRenderBundleEncoderPushDebugGroup", referenced from:
  "_wgpuRenderBundleEncoderSetBindGroup", referenced from:
  "_wgpuRenderBundleEncoderSetIndexBuffer", referenced from:
  "_wgpuRenderBundleEncoderSetPipeline", referenced from:
  "_wgpuRenderBundleEncoderSetVertexBuffer", referenced from:
  "_wgpuRenderPassEncoderBeginOcclusionQuery", referenced from:
  "_wgpuRenderPassEncoderBeginPipelineStatisticsQuery", referenced from:
  "_wgpuRenderPassEncoderEndOcclusionQuery", referenced from:
  "_wgpuRenderPassEncoderEndPipelineStatisticsQuery", referenced from:
  "_wgpuRenderPassEncoderExecuteBundles", referenced from:
  "_wgpuRenderPassEncoderInsertDebugMarker", referenced from:
  "_wgpuRenderPassEncoderPopDebugGroup", referenced from:
  "_wgpuRenderPassEncoderPushDebugGroup", referenced from:
  "_wgpuRenderPipelineGetBindGroupLayout", referenced from:
  "_wgpuRenderPipelineSetLabel", referenced from:
  "_wgpuShaderModuleGetCompilationInfo", referenced from:
  "_wgpuShaderModuleSetLabel", referenced from:```

**Comment by Beyley** [comment]
Getting this problem too with wgpuShaderModuleGetCompilationInfo, seems to still be a problem on the latest release?

**Comment by rajveermalviya** [comment]
Fixed in #202 & #246
